plugin-flex-ts-template-v2/src/feature-library/pause-recording/flex-hooks/events/pluginsInitialized.ts

```diff
@@ -0,0 +1,54 @@
+import * as Flex from '@twilio/flex-ui';
+import { Actions } from '@twilio/flex-ui';
+
+import { FlexEvent } from '../../../../types/feature-loader';
+import { isFeatureEnabled } from '../../config';
+import { registerPauseCallRecordingAction } from '../custom-action/pauseRecording';
+import { registerResumeCallRecordingAction } from '../custom-action/resumeRecording';
+import { registerToggleCallRecordingAction } from '../custom-action/toggleRecording';
+
+export const eventName = FlexEvent.pluginsInitialized;
+export const eventHook = function () {
+  if (!isFeatureEnabled()) return;
+
+  registerPauseCallRecordingAction();
+  registerResumeCallRecordingAction();
```

Comment:
I think the PauseRecordingButton component should be refactored to use these new actions so that everything is using the same code path.